packages/-ember-data/tests/integration/adapter/handle-response-test.js

```diff
@@ -175,4 +175,35 @@ module('integration/adapter/handle-response', function(hooks) {
 
     assert.equal(handleResponseCalled, 1, 'handle response is called');
   });
+
+  test('handleResponse is called with correct parameters on string repsonse with 422 status', async function(assert) {
```

Comment:
`assert.expect(3);`

Reply:
r/repsonse/response

Reply:
I don't get the assert.expect point, could you tell me more?

Reply:
Yeah!  So because you have assertions inside of stubbed methods that don't have to be called for the test to pass, by adding an `assert.expect`, we ensure that is is called.